Rakefile

```
@@ -54,42 +60,58 @@ def setup_analytics
   setup_formula_analytics_cmd
 end
 
-desc "Dump analytics data"
-task :analytics do
-  next unless generate_analytics?
-
-  setup_analytics
+def generate_analytics_files(os)
+  core_tap_name = os == "mac" ? "homebrew-core" : "linuxbrew-core"
+  analytics_data_path = "_data/analytics#{"-linux" if os == "linux"}"
```

```suggestion
  core_tap_name = "homebrew-core"
  analytics_data_path = "_data/analytics"
  os_suffix = nil
  if os == "linux"
    os_suffix = "--linux"
    analytics_data_path += os_suffix
  end
```

could also reuse that elsewhere. Liking using a single method here 👍

---

And we'd also need `core_tap_name` in the `if os == "linux"` conditional too, I think. Unless it doesn't matter what we call them?

---

Sorry, yes, missed that. Edited.